.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: ruby
+rvm:
+  - 1.9.3
```

Comment:
The only version I'm running currently (in windows) is 1.9.3. I'll need to update to 2.0.0 then. Have you tested the other versions?